modules/apps/transfer/keeper/relay.go

```diff
@@ -128,8 +129,9 @@ func (k Keeper) sendTransfer(
 			labels = append(labels, telemetry.NewLabel(coretypes.LabelSource, "false"))
 
 			// transfer the coins to the module account and burn them
-			if err := k.bankKeeper.SendCoinsFromAccountToModule(
-				ctx, sender, types.ModuleName, sdk.NewCoins(coin),
+			moduleAccountAddr := k.authKeeper.GetModuleAddress(types.ModuleName) // TODO: Change
+			if err := k.bankKeeper.SendCoins(
+				ctx, sender, moduleAccountAddr, sdk.NewCoins(coin),
```

Comment:
what's the purpose of this change?

Reply:
Yeah, I don't think this was actually needed, will remove